ingestion/src/metadata/ingestion/source/database/databricks/client.py

```diff
@@ -122,3 +126,78 @@ def is_query_valid(self, row) -> bool:
             query_text.startswith(QUERY_WITH_DBT)
             or query_text.startswith(QUERY_WITH_OM_VERSION)
         )
+
+    def list_jobs(self) -> List[dict]:
```

Comment:
Not sure if using `dict` as a typing here can have an impact on Python 3.7 🤔 I know we had some issues before with hints as `dict[...]`, so if you could please double-check that there won't be any hidden bug 🙏

Reply:
I did check the [Python 3.7 typing docs](https://docs.python.org/3.7/library/typing.html#typing.Dict). It seems that you need to use `typing.Dict` when you are also annotating the arguments, so like `Dict[str, str]`. So I think just `dict` without the annotated arguments should be fine.